src/db/external.id.db.gateway.ts

```diff
@@ -20,7 +20,7 @@ export class ExternalIdDbGateway {
      * This function will attempt to retrieve a single external IDs that corresponds to an id type and value. If the throwIfEmpty flag is set, then
      * the function will throw an error if there are no results.
      */
-    public fetchExternalId(externalIdType: string, externalIdValue: string, throwIfEmpty: boolean = true): Promise<ExternalId | undefined> {
+    public fetchExternalId(externalIdType: string, externalIdValue: string, throwIfEmpty = true): Promise<ExternalId | undefined> {
```

Comment:
why are we removing the type?  I know '= true' says its boolean but I believe we should be consistent in type declarations even when its "obvious"

Reply:
ESLint doesn't agree. Pretty sure this was done automatically via `eslint . --fix`.

I'm not sure I completely agree with the rule, but I think the transpiler assumes that, because we are assigning a boolean as a default, we want the parameter to be a boolean, and so functionally it comes out to the same thing.

Reply:
I'm fine with removing this. The transpiler catches the error if you provide a non-boolean value. It does feel strange to not have all parameters typed (and it feels dangerous for refactoring in case the default value goes away), but it's technically okay.

IMO If there's an easy way to change the rule, then we should. Otherwise, it's fine.

Reply:
Hoping that the linter will catch it if we don't type the parameter-without-default. The very least it could do for me.